mlir/unittests/IR/AttributeTest.cpp

```diff
@@ -483,4 +483,22 @@ TEST(CopyCountAttr, CopyCount) {
 #endif
 }
 
+// Test stripped printing using test dialect attribute.
+TEST(CopyCountAttr, PrintStripped) {
+  MLIRContext context;
+  context.loadDialect<test::TestDialect>();
+  // Doesn't matter which dialect attribute is used, just chose TestCopyCount
+  // given proximity.
+  test::CopyCount::counter = 0;
+  test::CopyCount copyCount("hello");
+  auto res = test::TestCopyCountAttr::get(&context, std::move(copyCount));
```

Comment:
```suggestion
  Attribute res = test::TestCopyCountAttr::get(&context, std::move(copyCount));
```

We need to type-erased here, to differentiate with a concrete attribute.

Reply:
Good point, done. (and I missed that I could have just hit the "commit suggestion" button here ...)